Examples/Detectors/TGeoDetector/include/ActsExamples/TGeoDetector/TGeoDetector.hpp

```diff
@@ -99,6 +100,8 @@ struct TGeoDetector : public ActsExamples::IBaseDetector {
       LayerTriplet<Options::Interval> zRange;
       LayerTriplet<double> splitTolR{0};
       LayerTriplet<double> splitTolZ{0};
+      LayerTriplet<std::vector<std::pair<int, Acts::BinningType>>> binning0;
+      LayerTriplet<std::vector<std::pair<int, Acts::BinningType>>> binning1;
```

Comment:
Can you add python bindings for these two added properties? I think that should be all that's needed on the python side.

Reply:
Like this? Examples/Python/src/Detector.cpp#L140-L141

Reply:
Yeah I think that should work.